### ISSUE TITLE
searchReferences only returns References with md5checksum == "TODO"

### ISSUE THREAD

**Issue by hjellinek** [ga4gh/server#539]
**Background**: I'm using data from the server unit tests as the server's data.  The reference data looks like this:
```
references/
    example_1/
        simple.fa.gz
        simple.fa.gz.fai
        simple.fa.gz.gzi
```

**Action**: I create a `SearchReferencesRequest` without setting the value of `md5checksums` (meaning it's equal to the empty list) and pass it to `searchReferences`.

**Expected**:
The call to `searchReferences` will return a `SearchReferencesResponse` with its `references`  containing a single `Reference` object with a valid `md5checksum` field (32 characters long, all hex, and ideally _correct_, though I have no means for testing the value's validity yet).

**Actual**:
The call to `SearchReferencesResponse` returns a `SearchReferencesResponse` containing a single `Reference`.  The `Reference`'s `md5checksum` field contains the string __`"TODO"`__.

**Comment by jeromekelleher** [comment]
This is related to #540. We can't compute MD5 values on the fly because references are large (several GB). Until we figure out how to properly import reference data, we need a filler value here. Is it better to have the empty string or something that looks like an md5 (say, 32 zeros)?

**Comment by jeromekelleher** [comment]
Fixed in #626.
